chart/compass/templates/migrator-job.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: v1
+data:
+    check-db-connection.sh: |
```

Comment:
why this is not part of the migrator image?

Reply:
Initially, I was thinking about using that script in init container running on alpine, now that it's used in migrator job, it makes sense to move it into that image.